package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@maxihost/metal-ui",
-  "version": "1.0.17",
+  "version": "1.0.18",
```

Comment:
I believe this shouldn't be versioned on PRs but rather we should create releases from the master branch once we have enough changes to warrant a new release, otherwise it'll keep conflicting with other PRs

Reply:
makes sense, let me change it.

Reply:
done